### PR TITLE
Use black formatting

### PR DESCRIPTION
**Pull request by ross-spencer**
Connected to archivematica/issues#393

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,3 +37,8 @@ matrix:
 
     - python: "3.6"
       env: TOXENV=py36-flake8
```
Comment (sevein) [inline .travis.yml]: py36-flake8 and py27-flake8 can be removed now, i.e.:

```yml
    - python: "2.7"
      env: TOXENV=py27-flake8

    - python: "3.6"
      env: TOXENV=py36-flake8
```

File: tox.ini [hunk 2]
```diff
@@ -17,7 +17,12 @@ commands = flake8 .
 
 [flake8]
 exclude = .git, .tox, __pycache__, old, build, dist
+application-import-names = flake8
```
Comment (sevein) [inline tox.ini]: `[testenv:py27-flake8]` and `[testenv:py36-flake8]` can be removed now.

File: tox.ini [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py27,py36}{,-flake8}
+envlist = {py27,py36}{,-flake8}, linting
```
Comment (sevein) [inline tox.ini]:
```suggestion
envlist = py27, py36, linting
```

File: .travis.yml (hunk 1 above)
Comment (ross-spencer) [reply .travis.yml]: Hi @sevein did you notice when you run tox with that newer setting then Flake8 doesn't run locally? That's often beneficial to the developer before pushing. I'm not sure though, maybe it's doing something different, or my configuration needs changing?

File: tox.ini (hunk 3 above)
Comment (ross-spencer) [reply tox.ini]: Hi @sevein did you notice when you run tox with this newer setting then Flake8 doesn't run locally? That's often beneficial to the developer before pushing. I'm not sure though, maybe it's doing something different, or my configuration needs changing?

Comment (sevein) [reply tox.ini]: I think that your `tox.ini` is missing the `linting` definition:

```ini
[testenv:linting]
basepython = python3
deps = pre-commit
commands = pre-commit run --all-files --show-diff-on-failure
```

Then:

```
$ tox -e linting
black....................................................................Passed
flake8...................................................................Failed
hookid: flake8
```

Comment (ross-spencer) [reply tox.ini]: :+1: that looks better now.

File: .travis.yml (hunk 1 above)
Comment (sevein) [reply .travis.yml]: This needs to go!

**Pull request merged**